src/applications/yellow-ribbon/containers/SearchResults/index.jsx

```diff
@@ -96,19 +106,53 @@ export class SearchResults extends Component {
     return endNumber - (perPage - 1);
   };
 
+  recordEventOnSearchResultClick = (school = {}) => () => {
+    // Derive the current name params.
+    const queryParams = new URLSearchParams(window.location.search);
+
+    // Derive the state values from our query params.
+    const name = queryParams.get('name') || '';
+
+    const { page, perPage, totalResults } = this.props;
+
+    return recordEvent({
```

Comment:
I think I commented this before but I might have missed your reply, but it's a little confusing to `return recordEvent` since `recordEvent` doesn't actually return anything.